filter-new-mean.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __FILTER_NEW_MEAN__
+#define __FILTER_NEW_MEAN__
+
+int filterNewMean()
+{
```

Comment:
I do not understand how this filter works
shouldnt that be just sum(data)/DATA_SIZE

Reply:
![newmean](https://user-images.githubusercontent.com/18075031/49136805-70e79500-f2f3-11e8-9d60-3369af558e2b.PNG)